src/main/java/uk/gov/hmcts/reform/cpo/payload/UpdateCasePaymentOrderRequest.java

```diff
@@ -47,4 +47,15 @@
         example = "Jane Doe")
     private String responsibleParty;
 
+    @NotNull(message = ValidationError.ORDER_REFERENCE_REQUIRED)
+    @Pattern(regexp = Validator.ORDER_REFERENCE_RG, message = ValidationError.ORDER_REFERENCE_INVALID)
+    @ApiModelProperty(value = "Description of the Payments system order reference", required = true,
```

Comment:
very minor comment, value could be constant as its used here and in CreateCasePaymentOrderRequest.java

Reply:
Sonar has pointed out the same duplication so I have resolved in c80421b353528a35e29094b40e05c56608035b19